src/services/PartsKit.php

```diff
@@ -33,14 +33,16 @@ public static function isRequest(): bool
      */
     public static function getConfig(string $key)
     {
+        // Merge user settings with the defaults
+        $userSettings = Craft::$app->config->getConfigFromFile('viget')['partsKit'] ?? [];
```

Comment:
Now that we are doing this in a number of places, I think it makes sense to move this into the module init, so we get the config from the file and merge with defaults only once.